objectcomputing/check-ins
# Feature 2111/convert team skills report to use transfer list

Closes #2111 

This feature introduces a new component, `MemberSelector`, which is a way to filter members by several criteria and select them. For this story, the component is being used for selecting members to use in a team skill report.

To test:
- Log in as an admin
- In the menu, go to REPORTS > Team Skills
- Add members via the member selector
  - Search by member name
  - Filter by guild, team, title, location, skill, or manager
  - Filtering by manager has the additional option of choosing only direct reports or all reports
- After selecting members, select skills for the report and click "Run Search"
- All functionality for the report results should be unchanged
